hazelcast/src/main/java/com/hazelcast/client/impl/spi/impl/ClientTransactionManagerServiceImpl.java

```diff
@@ -103,12 +102,12 @@ public void shutdown() {
     }
 
     @Override
-    public String getGroupName() {
-        final GroupConfig groupConfig = client.getClientConfig().getGroupConfig();
-        if (groupConfig == null) {
-            throw new RuntimeException("GroupConfig cannot be null client is participate in XA Transaction");
+    public String getClusterName() {
+        final String clusterName = client.getClientConfig().getClusterName();
+        if (clusterName == null) {
```

Comment:
how can a `clusterName` be `null`?

Reply:
I will remove this `if-block` which is a leftover from old code.